app/templates/layout.html.twig

```diff
@@ -9,12 +9,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     {% block extra_meta %}{% endblock %}
-    <link href="/css/bootstrap.min.css" rel="stylesheet">
+    <link href="/css/main.css" rel="stylesheet">
     <link href="/css/bootstrap-theme.min.css" rel="stylesheet">
     <link href="/css/font-awesome-custom.css" rel="stylesheet">
```

Comment:
Can't we include this file via SCSS as well? The less files need to be loaded the better…